packages/ovm/test/helpers.ts

```diff
@@ -105,7 +106,8 @@ export const executeUnsignedEOACall = async (
   executionManager: Contract,
   wallet: Wallet,
   to: Address,
-  data: string
+  data: string,
+  allowRevert: boolean
 ): Promise<TransactionReceipt> => {
```

Comment:
A function comment for `executeUnsignedEOACall` describing what each of the parameters is would be helpful! (Specifically about allowRevert)